openreview/conference/invitation.py

```diff
@@ -27,6 +28,9 @@ def __init__(self, conference, start_date, due_date, readers, additional_fields,
         for field in remove_fields:
             del content[field]
 
+        if isinstance(additional_fields, str):
+            additional_fields = json.loads(additional_fields)
```

Comment:
I would prefer to do this in the helpers function and the SubmissionInvitation should expect always a dictionary.